src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Button, Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import BasicInfo from "pages/Admin/BasicInfo";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueDetails from "pages/Admin/Venue/Details";
+
+import { Venue_v2 } from "types/venues";
+
+import "./AdminVenueView.scss";
+
+export interface SidebarOption {
+  id: string;
+  text: string;
+}
+
+export enum SidebarOptions {
+  dashboard = "dashboard",
+  basicInfo = "basic_info",
+  entranceExperience = "entrance_experience",
+  advancedMapSettings = "advanced_map_settings",
+  ticketingAndAccess = "ticketing_and_access",
+}
+
+const sidebarOptions: SidebarOption[] = [
+  {
+    id: SidebarOptions.basicInfo,
+    text: "Start",
+  },
+  {
+    id: SidebarOptions.entranceExperience,
+    text: "Entrance",
+  },
+  {
+    id: SidebarOptions.advancedMapSettings,
+    text: "Advanced",
+  },
+  {
+    id: SidebarOptions.dashboard,
+    text: "Dashboard",
+  },
+  // TODO: Reintroduce when field is decided what to include
+  // {
+  //   id: SidebarOptions.ticketingAndAccess,
+  //   text: "Ticketing and access",
+  // },
+];
+
+const DEFAULT_TAB = sidebarOptions.findIndex(
+  (option) => option.id === SidebarOptions.dashboard
+);
```

Comment:
Maybe it's better to create a `sidebarOptions` Map(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)? The structure would be [id: SidebarOptions] .You could iterate through Map.values() in the AdminVenueView component. Not 100% sure about it, but it looks like it would suit better. In particular I don't like finding the default value with findIndex. Looks unusual to me

Reply:
Even just a plain object would probably work fine in that use case. But then you would end up needing to reduce it to an array (or use `Object.values`/etc) again anyway.

The `.findIndex` code only runs once on app load, so it's not a 'persistent hit' to our performance (and presumably the `.values`/`.reduce`/etc would have to run in a similar way anyway)

Reply:
It's the approach that I find a bit hard to understand, but yeah, a regular object would work just fine